File: languages/kotlin/examples/Basic.kt
import org.positiontape.PositionTape

fun main() {
    val exact = PositionTape.Generate(100)
    val markerComplete = PositionTape.GenerateMarkerComplete(1000)
    val validation = PositionTape.Validate(exact, 100)

    println(exact)
    println(markerComplete.length)
    println(validation.isValid)
}

File: languages/dart/examples/basic.dart
import '../src/position_tape.dart' as pt;

void main() {
  final exact = pt.Generate(100);
  final markerComplete = pt.GenerateMarkerComplete(1000);
  final validation = pt.Validate(exact, 100);

  print(exact);
  print(markerComplete.length);
  print(validation.isValid);
}
